tests/gdimagestring16/gdimagestring16.c

```diff
@@ -9,9 +9,7 @@ int main()
 {
 	/* Declare the image */
 	gdImagePtr im = NULL;
-	wchar_t *wchr = L"H";
-	unsigned short *sptr;
-	sptr = (unsigned short *)wchr;
+  unsigned short s[2] = {'H', 0};
```

Comment:
Nit: inconsistent whitespace (tabs vs. spaces) (same for the other test below)

Reply:
Oh, caused by my vim configure